jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -750,7 +750,9 @@ Function Test-UserRegistryLoadState {
         # User Security Identifier
         [Parameter(Mandatory = $true)]
         [ValidatePattern("^S-\d-\d+-(\d+-){1,14}\d+$")]
-        [System.String]$UserSid
+        [System.String]$UserSid,
+        [Parameter(Mandatory = $false)]
+        [bool]$ValidateFolder
```

Comment:
Also I'm not sure there's a need to have a parameter here. I'm trying to think of one but not that we want this to be default behavior this seems unnecessary to have the choice to check redirection

Reply:
This was an optional task on the AC. I'm in favor of removing it actually.